modules/migration-using-must-gather.adoc

```diff
@@ -19,13 +27,33 @@ The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migra
 ----
 $ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version}
 ----
++
+The data is saved in a local `must-gather` directory.
+
 
-. Remove authentication keys and other sensitive information.
-. Create an archive file containing the contents of the `must-gather` data directory:
+
+. Optional: Collect an expanded set of metrics data by running the `oc adm must-gather` command with the `gather_metrics_dump` script:
 +
 [source,terminal]
 ----
-$ tar cvaf must-gather.tar.gz must-gather.local.<uid>/
+$ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version} -- /usr/bin/gather_metrics_dump
+----
++
+[NOTE]
+====
+This process might take a long time. `must-gather` processes the most recent `prom_data.tar.gz` file in the `/must-gather/metrics` directory.
+====
+. Optional: View the metrics data with Prometheus
+.. Create a local Prometheus instance to display the dumped data:
++
+[source,terminal]
+----
+$ make prometheus-run
+----
+.. In a browser, open the Prometheus web UI at http://localhost:9090.
```

Comment:
We do not use the term "UI". Use correct markup for the URL. See my MTV section.

Reply:
OK